charts/canvas-vault/Chart.yaml

```diff
@@ -15,13 +15,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.2.0
+# version: 0.2.0 - rollback to Vault version 1.14.8, which is the last under MPL
+# version: 0.1.0 - baseline version
 
 # This is the version number of the application being deployed. We are versioning the canvas as the
 # version of the latest component spec that it supports.
-appVersion: "0.28.0"
+appVersion: "1.14.8"
 
 dependencies:
   - name: vault
-    version: "0.28.0"
+    version: "0.26.1"
```

Comment:
Current chart version is 0.28.0, do we need to go back to 0.26.1?

Reply:
I looked for the last chart, which deployed app version 1.14.x. This was 0.25. 
So, the first try was to use 0.25, but this did not support namespace-override in the deployment, which was added in 0.26 which deploys Vault 1.15.x.
So, I updated the helm-chart to the latest 0.26 version 0.26.1 and downgraded all images to 1.14.8.